### PR TITLE
Add RSpec/NestedGroups cop

### PR DESCRIPTION
**Pull request by backus**

### CODE REVIEW

File: lib/rubocop-rspec.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@
 require 'rubocop/cop/rspec/instance_variable'
 require 'rubocop/cop/rspec/example_length'
 require 'rubocop/cop/rspec/multiple_describes'
+require 'rubocop/cop/rspec/nested_context'
```
Comment (andyw8) [inline lib/rubocop-rspec.rb]: Should be after `named_subject`.

**Comment by andyw8** [comment]
If I'm reading this correctly, it allows no nesting at all? I wonder if it should be configurable.

**Comment by backus** [comment]
> If I'm reading this correctly, it allows no nesting at all? I wonder if it should be configurable.

Yeah I agree

**Comment by andyw8** [comment]
Nice. I love this kind of cop – it's a very simple rule, but it can really help push code in the right direction since it highlights the smell of complex setup.

**Pull request merged**